### PR TITLE
Calculator operation functions added

### PR DESCRIPTION
**Pull request by sentayhu19**
**In this pull req:-**

- The files from the Calculator component Imported inside` Calculator.js`
-  Event handlers implemented
- Calculate.js and operate.js used
- Using `state` and `props`
- This app performs math operations

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,48 +1,63 @@
...
+        <button type="button" className="op" onClick={handleClick}>AC</button>
+        <button type="button" className="op" onClick={handleClick}>+/-</button>
+        <button type="button" className="op" onClick={handleClick}>%</button>
+        <button type="button" className="operator" onClick={handleClick}>÷</button>
+      </div>
+      <div id="row2">
+        <button type="button" className="number" id="7" onClick={handleClick}>7</button>
+        <button type="button" className="number" id="8" onClick={handleClick}>8</button>
+        <button type="button" className="number" id="9" onClick={handleClick}>9</button>
+        <button type="button" className="operator" onClick={handleClick}>*</button>
```
Comment (Mihndim2020) [inline src/components/Calculator.js]: - [x] Please the multiplication operator(`*`) has an issue, the app crashes each time I try to multiply two numbers.  Please checkout the console messages in the screenshot below and fix the issue.

multiplication_operator

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good job with Gitflow,
:heavy_check_mark: Your app is running free of `Linter errors`.

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please also consider:
- [OPTIONAL] updating your readme file with explicit steps to clone and run you app locally as this will be of help to non technical users.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js [hunk 2]
```diff
@@ -1,48 +1,63 @@
...
-        </div>
+      <div id="row4">
+        <button type="button" className="number" id="1" onClick={handleClick}>1</button>
+        <button type="button" className="number" id="2" onClick={handleClick}>2</button>
+        <button type="button" className="number" id="3" onClick={handleClick}>3</button>
+        <button type="button" className="operator" onClick={handleClick}>+</button>
+      </div>
+      <div id="row5">
+        <button type="button" id="n0" onClick={handleClick}>0</button>
+        <button type="button" className="number" onClick={handleClick}><b>.</b></button>
```
Comment (Gambit142) [inline src/components/Calculator.js]: - [x] (Optional) When I click, on the dot (.) button, it does not show concatenate as a string. This is what it shows:

image

I would like to advise you to remove the `<b>` tag and wrap the (.) in the button. 👍

**Review by Gambit142 — APPROVED** [review]
Hi @sentayhu19 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Optional Challenges
- Kindly implement the optional changes suggested under the review and the one suggested by the former reviewer. These optional suggestions make your project better and more presentable. 🙏 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**